hypha/apply/funds/forms.py

```diff
@@ -373,10 +381,14 @@ def submissions_cant_have_external_reviewers(self, submissions):
                 return True
         return False
 
-    def user_cant_alter_submissions_external_reviewers(self, submissions, user):
-        for submission in submissions:
-            if user != submission.lead and not user.is_superuser:
-                return True
+    def user_can_alter_submissions_external_reviewers(self, submissions, user):
+        # User needs to be superuser or lead of all selected submissions.
+        if user.is_superuser:
+            return True
+        if settings.GIVE_STAFF_LEAD_PERMS and user.is_apply_staff:
+            return True
+        if submissions.count() == submissions.filter(lead=user).count():
+            return True
```

Comment:
@frjo, I revamp this method a little..can you take a look?

cc: @sandeepsajan0

Reply:
Much more readable!